airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -8,22 +8,18 @@ tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
-      status: "excefailedption"
+      status: "failed"
   discovery:
     - config_path: "secrets/config.json"
   basic_read:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "sample_files/configured_catalog.json"
-# TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
-#      expect_records:
-#        path: "integration_tests/expected_records.txt"
-#        extra_fields: no
-#        exact_order: no
-#        extra_records: yes
-#  incremental: # TODO if your connector does not implement incremental sync, remove this block
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog.json"
-#      state_path: "integration_tests/abnormal_state.json"
+      configured_catalog_path: "integration_tests/product_catalog_v2_configured_catalog.json"
+      empty_streams:
+        - order
+  incremental:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/product_catalog_v2_configured_catalog.json"
+      future_state_path: "integration_tests/future_state.json"
   full_refresh:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "sample_files/configured_catalog.json"
+      configured_catalog_path: "integration_tests/product_catalog_v2_configured_catalog.json"
```

Comment:
same

Reply:
Right now we have access only to Product Catalog 2.0 and therefore we can test only streams for Product Catalog 2.0 which are: subscription, customer, invoice and order.
We don't have access to Product Catalog 1.0 and therefore we can't test plan and addons streams.

Reply:
Extra tests for what exactly? Right now we have access only to `Product Catalog 2.0` and therefore we can test only streams for `Product Catalog 2.0` which are: `subscription`, `customer`, `invoice` and `order`.
We don't have access to `Product Catalog 1.0` and therefore we can't test `plan` and `addons` streams.